test/functional/unified-spec-runner/operations.ts

```diff
@@ -173,11 +173,31 @@ operations.set('assertSessionTransactionState', async ({ entities, operation })
   expect(session.transaction.state).to.equal(driverTransactionStateName);
 });
 
+operations.set('assertNumberConnectionsCheckedOut', async ({ entities, operation }) => {
+  const client = entities.getEntity('client', operation.arguments.client);
+  const servers = Array.from(client.topology.s.servers.values());
+  const checkedOutConnections = servers.reduce((count, server) => {
+    const pool = server.s.pool;
+    return count + pool.currentCheckedOutCount;
+  }, 0);
+  expect(checkedOutConnections).to.equal(operation.arguments.connections);
+});
+
 operations.set('bulkWrite', async ({ entities, operation }) => {
   const collection = entities.getEntity('collection', operation.object);
   return collection.bulkWrite(operation.arguments.requests);
 });
 
+operations.set('close', async ({ entities, operation }) => {
+  try {
+    const cursor = entities.getEntity('cursor', operation.object);
+    await cursor.close();
```

Comment:
Not your doing, but this method claims to not return a promise, looks like the zero argument overload `close(): void` should be `close(): Promise<void>`, can we fix here?

Reply:
Yes. I've updated.